vmware_aria_operations_integration_sdk/project.py

```diff
@@ -199,3 +280,20 @@ def _add_and_update_project_paths(path: str) -> None:
             )
         ),
     )
+
+
+def _safe_append_to_gitignore(gitignore_file_path: str, token: str) -> None:
+    try:
+        with open(gitignore_file_path, "r") as gitignore:
+            for line in gitignore.readlines():
+                if token == line.rstrip("\n"):
```

Comment:
would `strip()` be more appropriate here? I'd imagine ignoring any whitespace would be ideal.